### ISSUE TITLE
PayPal Plus not working under certain circumstances

### ISSUE THREAD

**Issue by vstuber** [leadingsystems/contao-merconis#22]
The new checkout mostly works without actual page reloads and only loads the parts that need to be updated via Ajax. When a payment option is selected, the "payment selector" is reloaded via Ajax. If PayPal Plus is not displayed before but after the Ajax reload, it is necessary that a JS code that is being loaded together with the HTML code is being executed. With the new version of LSJS this is possible. However, PayPal Plus needs to load https://www.paypalobjects.com/webstatic/ppplus/ppplus.min.js and execute other JS code only after this library has been loaded completely. For reasons we don't fully understand yet, loading this library doesn't seem to work. Therefore, it is necessary to put the following code in the custom head section in the Contao layout to make sure that the PayPal Plus JS library is always available:
`<script src="https://www.paypalobjects.com/webstatic/ppplus/ppplus.min.js"></script>`

There's another issue, though: If PayPal Plus is selected and everything ist fine we can create a little problem by emptying the cart, closing it, adding another item to the cart and then opening the cart again. In this case, PayPal Plus is still selected but it doesn't show the PayPal Plus payment wall, unless we click on the PayPal Plus option again!

**Comment by chiron501** [comment]
PayPal has discontinued PayPal Plus
